src/index.js

```diff
@@ -20,5 +20,6 @@ export * from './components/ConfigurationSummary';
 // helpers
 export * from './constants';
 export * from './utils';
+export * from './models';
```

Comment:
Why do you want to export models ? We usually have all models in `web-ui` and `web-ui-components` contains only copies of necessary models from `web-ui`

Reply:
The TODO comment in `kubevirt/modes/vm.ts` [1] suggest reusing the models from web-ui-components. Exporting models here allows doing that. If I am testing it correctly, then I just need to re-export the model in `metalkube/models/host.ts` which I find better than maintaining 2 copies in 2 repositories.

Reply:
makes sense :)